trunk/SUAVE/Methods/Geometry/Two_Dimensional/Cross_Section/Airfoil/compute_airfoil_polars.py

```diff
@@ -149,7 +148,30 @@ def compute_airfoil_polars(a_geo,a_polar):
             # Pack this loop
             CL[i,j,:] = CL_ij
             CD[i,j,:] = CD_ij
-           
+            
+            if use_pre_stall_data == True:
```

Comment:
I think this should be a separate function. The reason is that AERODAS for aerodynamics and for propellers can both use this code without duplication